### ISSUE TITLE
Add description for containers

### ISSUE THREAD

**Issue by xiaojinghu93** [singularityhub/sregistry#381]
Hi, all,
We've deployed sregistry in our clusters, of which the  software and hardware environments are  both quite complex.  We'd like to add descriptions to containers to help users/developers grasp  infos  like the architecture, the version, the compiler and so on.

Sregistry's already provided a separate page for each container. It would be nice if the description can be displayed and edited on that page. Currently we have to differentiate the containers by name(with long suffix).

**Comment by vsoch** [comment]
Yep, can definitely add this! And likely we can store with the container metadata so the database doesn’t need a migration. And you’d be ok with editing the interface to add the description?

mine more suggestion I’ll make in terms of labels is to use org.opencontainers.image.description - it’s currently used by GitHub packages containers for the same goal.

I should have some time to work on this soon.

**Comment by vsoch** [comment]
okay, please review https://github.com/singularityhub/sregistry/pull/382 !
